pdf/parser-polytracker-daedalus/config.json5

```diff
@@ -0,0 +1,43 @@
+{
+  parsers: {
+    "daedalus-file-cavities": {
+      version: 'polytracker-v1.2',
+      exec: ['python3', 'main.py', '<inputFile>'],
+      timeout: 60,
+      parse: {
+        type: 'regex-counter',
+        version: '1.7',
+        stdstar: {
+          '^"(([^"\\\\]|\\\\.)*?)","(([^"\\\\]|\\\\.)*?)","': {
+            nameGroup: 'INFIX: "\\g<3>"',
+            nameReplace: {
+              '[0-9]+': '<INT>',
+            },
+            fallthrough: true,
+          },
+          '^.*$': {
+            nameReplace: {'[0-9]+': '<INT>'},
+          }
+        },
+      },
+    },
+  },
+  build: {
+    stages: {
+      mupdf: {
+        from: 'trailofbits/polytracker:faw-daedalus-2022-07-27',
```

Comment:
It seems like the steps for building this image should be here, rather than relying on a pre-built from ToB? Otherwise, updating e.g. the version of DaeDaLus is outside of the FAW's control, which could become a problem.

Reply:
E.g., the `commands` array shouldn't be empty here, but should apply polytracker to daedalus, producing the below binaries. If possible.

Reply:
This was a historical issue, because the relevant code was on a branch of PolyTracker. This has since been edited.